libraries/testing/tester.cpp

```diff
@@ -246,14 +251,6 @@ namespace eosio { namespace testing {
       }
 
       if( !skip_pending_trxs ) {
-         unapplied_transactions_type unapplied_trxs = control->get_unapplied_transactions(); // make copy of map
```

Comment:
It concerns me that `_produce_block` no longer re-applies unapplied transactions, even though it apparently does not break any of our existing unit tests. I would like tester to track the unapplied transactions using `unapplied_transaction_queue` similar to how producer plugin does, so that it can re-apply them here.

Reply:
Changed tester to use unapplied_transaction_queue